www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+- Adding your first Svelte component
+- Importing images and other web assets
+- Enabling Hot Module Replacement (HMR)
+- Connecting your favorite tools
+
+Prerequisites: Snowpack is a command-line tool installed from npm. This guide assumes a basic understanding of Node.js, npm, and how to run commands in the terminal. Knowledge of Svelte is not required, Snowpack is an excellent way to learn Svelte!
+
+> 💡 Tip: a [Svelte/Snowpack](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/app-template-svelte) working example is available in our Create Snowpack App templates.
+
+## Getting started
+
+The easiest way to start a new Snowpack project is with [Create Snowpack App](<https://www.snowpack.dev/#create-snowpack-app-(csa)>), a tool for creating a new project based on our example templates. `@snowpack/app-template-minimal` is a Create Snowpack App template for a simple, bare-bones Snowpack project setup that the rest of this guide builds on.
+
+Run the following command in your terminal to create a new directory called `svelte-snowpack` with the minimal template installed:
+
+```bash
+npx create-snowpack-app svelte-snowpack --template @snowpack/app-template-minimal
+```
+
+Head to the new `svelte-snowpack` directory and start Snowpack with the following two commands:
+
+```bash
+cd svelte-snowpack
+npm run start
+```
+
+You should see your new website up and running!
+
+<div class="frame"><img src="/img/guides/react/minimalist-hello-world.png" alt="screenshot of project-template-minimal, which shows 'Hello world' in text on a white background." class="screenshot"/></div>
+
+Now that you have a basic project up and running! The next step is to install Svelte. Run the following command in your project directory:
+
+```bash
+npm install svelte --save
+```
+
+> 💡 Tip: add the "--use-yarn" or "--use-pnpm" flag to use something other than npm
+
+Snowpack [plugins](/plugins) are a way to extend Snowpack's capabilities without having to do custom configuration yourself. Install the `@snowpack/plugin-svelte` plugin so that Snowpack knows how built `.svelte` files into JavaScript and CSS files that run in the browser:
+
+```bash
+npm install @snowpack/plugin-svelte --save-dev
+```
+
+Once installed, you'll need to add the plugin to your Snowpack configuration file (`snowpack.config.js`) so that Snowpack knows to use it:
+
+```diff
+// snowpack.config.js
+
+module.exports = {
+  mount: {
+    /* ... */
+  },
+  plugins: [
+-    /* ... */
++    '@snowpack/plugin-svelte'
+  ],
+```
+
+Restart your Snowpack dev server to run it with the new configuration. Exit the process (ctrl + c in most Windows/Linux and cmd + c for Mac) and start it again with `npm run start`.
+
+> 💡 Tip: Restart the Snowpack development server when you make configuration changes (changes to the `snowpack.config.js`).
+
+The Snowpack Development server will show the following as it recognizes your new dependency (Svelte) and turns it into ESM:
```

Comment:
"turns it into ESM" may be overwhelming given that we haven't explained that concept yet. Maybe just:

> Snowpack will recognize the new new dependency (Svelte, or "svelte/internal") and print the following output as installs your dependencies for the frontend: